contracts/test/strategies/curvePoolBooster.mainnet.fork-test.js

```diff
@@ -0,0 +1,427 @@
+const { expect } = require("chai");
+const { parseUnits } = require("ethers/lib/utils");
+const { impersonateAndFund } = require("../../utils/signers.js");
+
+const addresses = require("../../utils/addresses");
+const { isCI } = require("../helpers");
+
+const { loadDefaultFixture } = require("../_fixture");
+
+describe("ForkTest: CurvePoolBooster", function () {
+  this.timeout(0);
+
+  // Retry up to 3 times on CI
+  this.retries(isCI ? 3 : 0);
+
+  let fixture;
+  beforeEach(async () => {
+    fixture = await loadDefaultFixture();
+  });
+
+  it("Should have correct params", async () => {
+    const { curvePoolBooster } = fixture;
+    const { strategistAddr } = await getNamedAccounts();
+    expect(await curvePoolBooster.gauge()).to.equal(
+      addresses.mainnet.CurveOUSDUSDTGauge
+    );
+    expect(await curvePoolBooster.campaignRemoteManager()).to.equal(
+      addresses.mainnet.CampaignRemoteManager
+    );
+    expect(await curvePoolBooster.rewardToken()).to.equal(
+      addresses.mainnet.OUSDProxy
+    );
+    expect(await curvePoolBooster.targetChainId()).to.equal(42161);
+    expect(await curvePoolBooster.strategistAddr()).to.equal(strategistAddr);
+    expect(await curvePoolBooster.governor()).to.equal(
+      addresses.mainnet.Timelock
+    );
+  });
+
+  it("Should Create a campaign", async () => {
+    const { curvePoolBooster, ousd, wousd } = fixture;
+    const { strategistAddr } = await getNamedAccounts();
+    const woethSigner = await impersonateAndFund(wousd.address);
+    const sStrategist = await ethers.provider.getSigner(strategistAddr);
+
+    // Deal OETH and ETH to pool booster
```

Comment:
below 20 lines get repeated 4 times in the test file. Separating it into a function will improve the readability